diff --git a/src/lib/layout-utils.js b/src/lib/layout-utils.js
index 8dbf2819747b555b12b612c707f02681a3c8902d..92ee9adb6b61aad75e6792fae1001fb3a43f8335 100644
--- a/src/lib/layout-utils.js
+++ b/src/lib/layout-utils.js
@@ -1,5 +1,5 @@
 import fs from 'fs/promises';
-import htmlparser from 'node-html-parser';
+import * as htmlparser from 'node-html-parser';
 import { checkResourceExists } from './resource-utils.js';
 import { Worker } from 'worker_threads';
 
diff --git a/src/lib/resource-utils.js b/src/lib/resource-utils.js
index 6a78e464785b6dc11e909acf64f66db975e77909..1720b6e2f6692acfbe2db3ab31b86fbbeae198ed 100644
--- a/src/lib/resource-utils.js
+++ b/src/lib/resource-utils.js
@@ -1,6 +1,6 @@
 import fs from 'fs/promises';
 import { hashString } from './hashing-utils.js';
-import htmlparser from 'node-html-parser';
+import * as htmlparser from 'node-html-parser';
 
 async function modelResource(context, type, src = undefined, contents = undefined, optimizationAttr = undefined, rawAttributes = undefined) {
   const { projectDirectory, scratchDir, userWorkspace } = context;
diff --git a/src/plugins/resource/plugin-standard-html.js b/src/plugins/resource/plugin-standard-html.js
index 06223cf3dea165330e1b77e241d94f8c6b1a8c86..9c6d712083cf344f5186e64d3e6d6a235774aab3 100644
--- a/src/plugins/resource/plugin-standard-html.js
+++ b/src/plugins/resource/plugin-standard-html.js
@@ -17,7 +17,7 @@ import { getUserScripts, getPageLayout, getAppLayout } from '../../lib/layout-ut
 import { requestAsObject } from '../../lib/resource-utils.js';
 import unified from 'unified';
 import { Worker } from 'worker_threads';
-import htmlparser from 'node-html-parser';
+import * as htmlparser from 'node-html-parser';
 
 class StandardHtmlResource extends ResourceInterface {
   constructor(compilation, options) {
