diff --git a/src/lib/layout-utils.js b/src/lib/layout-utils.js
index fc5548ec78b28d12ae0602188ccbb03d288b3ce4..d592abe80bb146db4a1ee8429584aa82f2448f29 100644
--- a/src/lib/layout-utils.js
+++ b/src/lib/layout-utils.js
@@ -1,5 +1,5 @@
 import fs from "fs/promises";
-import htmlparser from "node-html-parser";
+import * as htmlparser from "node-html-parser";
 import { checkResourceExists } from "./resource-utils.js";
 import { Worker } from "worker_threads";
 import { asyncFilter } from "./async-utils.js";
diff --git a/src/lib/resource-utils.js b/src/lib/resource-utils.js
index 0fc513d8af755ff888f05cc87ac6eda9a64128e4..0d9e0a4f19349c5e963bc3b9216c0f7fa385f2da 100644
--- a/src/lib/resource-utils.js
+++ b/src/lib/resource-utils.js
@@ -1,7 +1,7 @@
 import fs from "fs/promises";
 import { hashString } from "./hashing-utils.js";
 import { getResolvedHrefFromPathnameShortcut } from "../lib/node-modules-utils.js";
-import htmlparser from "node-html-parser";
+import * as htmlparser from "node-html-parser";
 import { asyncMap } from "./async-utils.js";
 
 async function modelResource(
diff --git a/src/plugins/resource/plugin-standard-html.js b/src/plugins/resource/plugin-standard-html.js
index 0ed408965b55c8c4aa9e083a5e9cc26c12f13b23..aaf387fbabe7ea8fd647c9268c19baed39d6dd01 100644
--- a/src/plugins/resource/plugin-standard-html.js
+++ b/src/plugins/resource/plugin-standard-html.js
@@ -15,7 +15,7 @@ import { getUserScripts, getPageLayout, getAppLayout } from "../../lib/layout-ut
 import { requestAsObject } from "../../lib/resource-utils.js";
 import { unified } from "unified";
 import { Worker } from "worker_threads";
-import htmlparser from "node-html-parser";
+import * as htmlparser from "node-html-parser";
 
 class StandardHtmlResource extends ResourceInterface {
   constructor(compilation, options) {
